tests/modules/nf-core/panacus/histgrowth/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PANACUS_HISTGROWTH } from '../../../../../modules/nf-core/panacus/histgrowth/main.nf'
+
+workflow test_panacus_histgrowth {
```

Comment:
My wild guess here is that this is not in accordance with the new nf-test rules, but we will find out once you make this a real PR.